### PR TITLE
Integrating with toolkit

### PR DESCRIPTION
**Pull request by ktp242**
This PR integrates with NYPL design toolkit to give styles following our NYPL design pattern. Mainly, it gives the result table the styles to help the users read the information easier. It also hide the previous button or next button based on which page the user is on. Finally, it updates the date inputs from text type to date type.

This PR also solves part of the accessibility issues, however, using date input leads another issue to voice over that the inputs can't be read. A ticket will be created addressing on that. I will consult with Willa to see how I can fix it.

Can be test on dev here
http://nypl-recap-admin-dev.us-east-1.elasticbeanstalk.com/

### CODE REVIEW

**Review by ricardoom — COMMENTED** [review]
after talking w/ @ktp242 we need to review usage of using an HTML5 date input field. Its supported by most browserss _except_ Safari. Additionally the pager buttons under the Refile error table, isnt working in Firefox... but _does_ work in Webkit browsers...

**Comment by ricardoom** [comment]
Otherwise its looking good.

**Comment by ktp242** [comment]
I will fix the button in this PR, but can we leave the usage of date input fields for a later one? @thisisstephenbetts @ricardoom

**Comment by ktp242** [comment]
Ok I fixed the FF issue! Please check here! @ricardoom 
http://nypl-recap-admin-dev.us-east-1.elasticbeanstalk.com/refile

But be aware that the branch on the dev server is later than this PR, so it contains some other features that this PR doesn't have.

File: config/appConfig.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default {
   },
   nyplMicroService: {
     tokenUrlForNyplApiClient: process.env.TOKEN_URL_FOR_NYPL_API_CLIENT,
-    platformBaseUrl: process.env.NODE_ENV === 'production' ? process.env.PLATFORM_BASE_URL : process.env.DEV_PLATFORM_BASE_URL,
+    platformBaseUrl: process.env.APP_ENV === 'production' ? process.env.PLATFORM_BASE_URL : process.env.DEV_PLATFORM_BASE_URL,
```
Comment (thisisstephenbetts) [inline config/appConfig.js:25]: `APP_ENV` sounds bette to me — thanks!

File: src/server/routes/api/index.js [hunk 2]
```diff
@@ -119,27 +119,27 @@ function constructDateQuery(dateInput, isEndDate = false) {
   const lastSecond = (isEndDate) ? 'T23:59:59' : '';
 
   if (dateInput && typeof dateInput === 'string') {
-    const dateArray = dateInput.split('/');
+    const dateArray = dateInput.split('-');
 
     // Checks if it has a valid date format. The Regex check if the inputs are digits
     // and if they have right number of digits
-    const date_matches = dateInput.match(/^(\d{2})\/(\d{2})\/(?:\d{4})$/);
+    const date_matches = dateInput.match(/^(\d{4})\-(\d{2})\-(?:\d{2})$/);
```
Comment (thisisstephenbetts) [inline src/server/routes/api/index.js:128]: Nice!

File: src/server/routes/api/index.js [hunk 3]
```diff
@@ -119,27 +119,27 @@ function constructDateQuery(dateInput, isEndDate = false) {
...
 
     if (!date_matches) {
       return undefined;
     }
 
     // Checks if the month is valid
-    if (parseInt(dateArray[0], 10) < 1 || parseInt(dateArray[0], 10) > 12) {
+    if (parseInt(dateArray[1], 10) < 1 || parseInt(dateArray[1], 10) > 12) {
```
Comment (thisisstephenbetts) [inline src/server/routes/api/index.js]: Ha ha! See, if you'd had `month` and `year` as variables you could have saved 2 or 3 keystrokes ;-)

Comment (thisisstephenbetts) [reply src/server/routes/api/index.js]: [Callback to previous PR in case anyone else is confused]

**Review by thisisstephenbetts — APPROVED** [review]
Can't swear that I read every-line of the react components & CSS, etc., but all code looks good to me.

File: src/server/routes/api/index.js (hunk 3 above)
Comment (ktp242) [reply src/server/routes/api/index.js]: Updated it in the previous PR haha

**Pull request merged**